sklearn/utils/multiclass.py

```diff
@@ -167,20 +159,8 @@ def is_label_indicator_matrix(y):
                                     _is_integral_float(labels))
 
 
-def is_sequence_of_sequences(y):
-    """ Check if ``y`` is in the sequence of sequences format (multilabel).
-
-    This format is DEPRECATED.
-
-    Parameters
-    ----------
-    y : sequence or array.
-
-    Returns
-    -------
-    out : bool,
-        Return ``True``, if ``y`` is a sequence of sequences else ``False``.
-    """
+def _is_sequence_of_sequences(y):
```

Comment:
If it is only used one, I would inline the function and add appropriate comment.